packages/patrol/android/src/main/kotlin/pl/leancode/patrol/BrowserstackPatrolJUnitRunner.java

```diff
@@ -38,7 +38,7 @@ public String getLoopback() {
                 if (Objects.equals(i.getDisplayName(), "tun0")) {
                     for (java.net.InterfaceAddress a : i.getInterfaceAddresses()) {
                         if (a.getAddress() instanceof Inet4Address) {
-                            return a.getAddress().toString().substring(1) + ":8082";
+                            return a.getAddress().toString().substring(1);
```

Comment:
why this change?

Reply:
We pass the port to the client as a separate argument, we don't need to concatenate it here.